kolibri/core/assets/src/views/sync/SelectAddressModalGroup/SelectAddressForm.vue

```diff
@@ -237,6 +244,14 @@
         return null;
       },
     },
+    watch: {
+      addresses(addresses) {
+        this.availableAddressIds = addresses
+          .filter(address => address.available || address.hasContent)
```

Comment:
One small thing here, I haven't tested it yet, but I think this should be `a.available && a.hasContent`, since `!(!a.available || !a.hasContent)` is the same as `a.available && a.hasContent`.

In other words, we want to consider the address to be "available" if it both has `a.available == true` and `a.hasContent == true`.

Reply:
Actually, the `hasContent` condition doesn't need to hold if you're only syncing Facility data. Maybe it's okay if the address is merely available. If the user is trying to import content, they will be taken to a page with no channels, which is okay.

Reply:
I think it's okay to just remove the `|| address.hasContent` in the predicate, because I don't want this modal to block us from syncing with devices that might only have Facility/User data but no content.